### PR TITLE
Allow support PreferenceFragment with @PreferenceScreen

### PR DESCRIPTION
**Pull request by WonderCsabo**
Implements #1396.

I decided to support two unofficial libraries.

### CODE REVIEW

File: AndroidAnnotations/androidannotations/src/test/java/org/androidannotations/preference/SettingsFragment.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.androidannotations.preference;
+
+import org.androidannotations.annotations.EFragment;
+import org.androidannotations.annotations.PreferenceScreen;
+
+import android.preference.PreferenceFragment;
```
Comment (dodgex) [inline AndroidAnnotations/androidannotations/src/test/java/org/androidannotations/preference/SettingsFragment.java]: shouldn't that be `android.support.v4.preference.PreferenceFragment`?

Comment (dodgex) [inline AndroidAnnotations/androidannotations/src/test/java/org/androidannotations/preference/SettingsFragment.java]: ok nevermind. just saw that the next file had the support version.

**Comment by yDelouis** [comment]
I'm OK with the code. I can't test it right now. If you're confident, you can merge this PR.

**Comment by WonderCsabo** [comment]
Rebased and merged as of a4c7488.

**Pull request closed without merging**